src/MeetingsSDKAdapter.js

```diff
@@ -936,13 +936,21 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
 
       const meetingWithRosterToggleEvent$ = fromEvent(sdkMeeting, EVENT_ROSTER_TOGGLE);
 
+      const meetingStateChange$ = fromEvent(sdkMeeting, EVENT_STATE_CHANGE)
+        .pipe(
```

Comment:
pls move this line (pipe) to the line above, like the other observables (line 922, line 926)